components/clarinet-files/src/project_manifest.rs

```diff
@@ -257,6 +305,38 @@ impl ProjectManifest {
             }
         };
 
+        let mut override_boot_contracts_source = BTreeMap::new();
+        if let Some(overrides) = project_manifest_file.project.override_boot_contracts_source {
+            for (contract_name, contract_path) in overrides.iter() {
+                override_boot_contracts_source.insert(contract_name.clone(), contract_path.clone());
+            }
+        }
+
+        let boot_contracts = vec![
+            "costs".to_string(),
+            "pox".to_string(),
+            "pox-2".to_string(),
+            "pox-3".to_string(),
+            "pox-4".to_string(),
+            "lockup".to_string(),
+            "costs-2".to_string(),
+            "costs-3".to_string(),
+            "cost-voting".to_string(),
+            "bns".to_string(),
+        ];
```

Comment:
```suggestion
        let boot_contracts = BOOT_CONTRACTS_NAMES
            .iter()
            .map(|n| n.to_string())
            .collect::<Vec<_>>();
```

With `use clarity_repl::repl::boot::BOOT_CONTRACTS_NAMES;`